src/richie/apps/courses/templates/courses/cms/person_detail.html

```diff
@@ -10,14 +10,21 @@ <h1 class="person-detail__title">
     {% with header_level=2 %}
       <div class="person-detail__card__media">
         {% with width=216 height=120 %}
-          {% placeholder "portrait" %}
+          {% placeholder "portrait" or %}
+            <div class="person-detail__card__media__empty">{% trans "Portrait" %}</div>
+          {% endplaceholder %}
         {% endwith %}
       </div>
 
       <div class="person-detail__card__content">
         <div class="person-detail__card__content__wrapper">
-        {% placeholder "resume" %}
-        {% placeholder "maincontent" %}
+          {% placeholder "resume" or %}
+            <p class="person-detail__card__content__wrapper__empty">{% trans "Resume" %}</p>
```

Comment:
Maybe a sentence like "Enter here your bio here" ?  Resume means CV which is not what we want here?